src/main/java/domain/board/Board.java

```diff
@@ -43,38 +49,54 @@ public List<Piece> extractPathPiece(List<BoardLocation> allPath) {
                 .toList();
     }
 
+    public Score calculateScoreByTeam(Team team) {
+        return pieces.keySet().stream()
+                .filter(location -> pieces.get(location).getTeam() == team)
+                .map(location -> pieces.get(location).getScore())
+                .reduce(Score::plus)
+                .orElse(new Score(0));
```

Comment:
대다수의 pieces에서 keySet을 호출해서 처리하신 부분들이 대다수는 values()로 value만 꺼내와서 처리할 수 있는 것 같아요. 🤔

```suggestion
       return pieces.values()
            .stream()
            .filter(it -> it.getTeam() == team)
            .map(Piece::getScore)
            .reduce(Score::plus)
            .orElse(new Score(0));
```

Reply:
map이 제공해주는 함수를 잘 숙지하지 못했던 것 같습니다!
values()를 적용하며 더 깔끔해진 것 같아요! 감사합니다